internal/controller/controller.go

```diff
@@ -88,8 +88,7 @@ func Run(
 
 	queue := agentTags["queue"]
 	if queue == "" {
-		logger.Error("The 'queue' tag is missing, but it is required", zap.Error(err))
-		return
+		logger.Info("Listening to the default job queue, to listen to a specific queue, please pass in 'queue' tag")
```

Comment:
```suggestion
		logger.Info("Listening to the default queue for the given cluster. To listen to a specific queue, please pass in 'queue' tag")
```

also, might it be worth doing the defaulting here (`queue = "_default"`) here to keep all of the queue logic in the same place?

Reply:
Updated the log line. 

> might it be worth doing the defaulting here

hmm, I don't think we should make the `_default` concept visible beyond transport layer, since it's really a transport layer thing. 

For most code paths in agent stack k8s, it should probably operate as: "I didn't set a queue tag, I don't know about the _default string".